scripts/audioplot.py

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/python
+
+# audioplot is a simple script for drawing graphs of values.  It reads a simple
+# csv text file where each line needs to contain the same number of comma-separated values,
+# like so:
+# 0, 0, 0
+# 1, 2, 1
+# 2, 4, 2
+#
+# Each column will be plotted as a separate curve on the same time series.
+# Matplotlib is used to display the result.
+
+
+from matplotlib import pyplot as plt
+from matplotlib.lines import Line2D
+import time, string, glob, sys
+
+import getopt
```

Comment:
WTF I can't make a PR to your repository oO. Anyway this is the commit.

https://github.com/kain88-de/mixxx/commit/ee4024de4bdf702e56c2a20e7f92955dfd692c70